### ISSUE TITLE
[Bug] Incorrect storage usage

### ISSUE THREAD

**Issue by Zenderix** [MauriceNino/dashdot#108]
### Description of the bug

dash. is showing the wrong used storage in the dashboard. 
![image](https://user-images.githubusercontent.com/44755471/173132481-05906cf2-8026-45d2-bc2a-631149f60721.png)

df output shows the right used storage
![image](https://user-images.githubusercontent.com/44755471/173132554-8ab5a845-6df8-4c18-860f-d7235052f2c4.png)

it should be more like this, so a lot more than 2.5tb...
![image](https://user-images.githubusercontent.com/44755471/173132763-64a2718d-092b-4025-a8a0-43d75716b5bc.png)

my docker compose volumes:

      volumes:
        - /etc/os-release:/etc/os-release:ro
        - /proc/1/ns/net:/mnt/host_ns_net:ro
        - /media:/mnt/host_media:ro
        - /storage:/mnt/host_storage
        - /storage/torrents:/mnt/host_torrents
        - /mnt:/mnt/host_mnt:ro
        - 

### Where is your instance running?

Linux Server

**Comment by MauriceNino** [comment]
Hello, thank you for creating an issue! Is the `df` run inside the container, or outside? Because the mountpoints seem off to me.

**Comment by Zenderix** [comment]
It's run inside the container. I had some leftovers from trying out things in there, here is the current setup:
![image](https://user-images.githubusercontent.com/44755471/173141064-4a0dae8f-e645-4f7e-9f67-027f2917e77e.png)

**Comment by MauriceNino** [comment]
Can you please run the following command in your host and paste the output?

```
docker container exec -it dash. /bin/sh -c 'mkdir -p /test-storage && cd /test-storage && echo \'import * as si from "systeminformation";const run = async () => { const [disks, sizes] = await Promise.all([si.diskLayout(), si.fsSize()]); console.log("Disks", disks); console.log("Sizes", sizes);}; run();\' > index.js && echo \'{"name": "test-storage","version": "1.0.0","type": "module","description": "","main": "index.js","scripts": {},"author": "","license": "ISC","dependencies": {"systeminformation": "^5.11.16"}}\' > package.json && npm i && node .'
```

**Comment by Zenderix** [comment]
> Can you please run the following command in your host and paste the output?
> 
> ```
> docker container exec -it dash. /bin/sh -c 'mkdir -p /test-storage && cd /test-storage && echo \'import * as si from "systeminformation";const run = async () => { const [disks, sizes] = await Promise.all([si.diskLayout(), si.fsSize()]); console.log("Disks", disks); console.log("Sizes", sizes);}; run();\' > index.js && echo \'{"name": "test-storage","version": "1.0.0","type": "module","description": "","main": "index.js","scripts": {},"author": "","license": "ISC","dependencies": {"systeminformation": "^5.11.16"}}\' > package.json && npm i && node .'
> ```

-bash: syntax error near unexpected token `('

**Comment by MauriceNino** [comment]
Doesn't work on bash as it seems...

Try this instead:

```
docker container exec -it dash. /bin/sh -c 'git clone https://github.com/MauriceNino/disk-info /disk-info && cd /disk-info && npm i && node .'
```

**Comment by Zenderix** [comment]
> Doesn't work on bash as it seems...
> 
> Try this instead:
> 
> ```
> docker container exec -it dash. /bin/sh -c 'git clone https://github.com/MauriceNino/disk-info /disk-info && cd /disk-info && npm i && node .'
> ```

/bin/sh: git: not found

ungute gschicht xD

**Comment by MauriceNino** [comment]
damn git is already installed in the dev version - last try haha:

```
docker container exec -it dashdot_dev_dash_1 /bin/sh -c 'apk add git && git clone https://github.com/MauriceNino/disk-info /disk-info && cd /disk-info && npm i && node .'
```

**Comment by Zenderix** [comment]
> damn git is already installed in the dev version - last try haha:
> 
> ```
> docker container exec -it dashdot_dev_dash_1 /bin/sh -c 'apk add git && git clone https://github.com/MauriceNino/disk-info /disk-info && cd /disk-info && npm i && node .'
> ```

Disks [
  {
    device: '/dev/sda',
    type: 'HD',
    name: 'ST1000LM024 HN-M',
    vendor: 'Seagate',
    size: 1000204886016,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '0001',
    serialNum: '',
    interfaceType: 'SATA',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/sdb',
    type: 'HD',
    name: 'EARS-00Y5B1     ',
    vendor: 'WDC WD10',
    size: 1000204886016,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '0327',
    serialNum: '',
    interfaceType: 'USB',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/sdc',
    type: 'HD',
    name: 'EZEX-08WN4A0    ',
    vendor: 'WDC WD10',
    size: 1000204886016,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '0327',
    serialNum: '',
    interfaceType: 'USB',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/sdd',
    type: 'HD',
    name: '000-1H4168      ',
    vendor: 'ST4000VN',
    size: 4000787030016,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '0327',
    serialNum: '',
    interfaceType: 'USB',
    smartStatus: 'unknown',
    temperature: null
  }
]
Sizes [
  {
    fs: 'overlay',
    type: 'overlay',
    size: 527295578112,
    used: 44320841728,
    available: 456114413568,
    use: 8.86,
    mount: '/'
  },
  {
    fs: '/dev/sda2',
    type: 'ext4',
    size: 527295578112,
    used: 44320841728,
    available: 456114413568,
    use: 8.86,
    mount: '/mnt/host_media'
  },
  {
    fs: '/dev/sdd1',
    type: 'ext4',
    size: 3936818806784,
    used: 2702606073856,
    available: 1034156703744,
    use: 72.32,
    mount: '/mnt/host_storage'
  },
  {
    fs: '/dev/mapper/torrents-torrents',
    type: 'ext4',
    size: 1967854387200,
    used: 1662309072896,
    available: 205508231168,
    use: 89,
    mount: '/mnt/host_storage/torrents'
  }
]

wouldn't be any fun if it was easy :P

**Comment by MauriceNino** [comment]
Can you try pulling the latest dev image and running that instead of prod? (`docker image pull mauricenino/dashdot:dev`)

**Comment by Zenderix** [comment]
> Can you try pulling the latest dev image and running that instead of prod? (`docker image pull mauricenino/dashdot:dev`)

seems to work perfectly now!

![image](https://user-images.githubusercontent.com/44755471/173147408-c5df6225-f2e7-4ad8-9e79-2f3e23673a77.png)


Besten Dank!

**Comment by MauriceNino** [comment]
Kein Problem ;)

New main release will be published tomorrow, if you want to switch back from dev, btw.

**Comment by Zenderix** [comment]
> Kein Problem ;)
> 
> New main release will be published tomorrow, if you want to switch back from dev, btw.

If I don't forget I'll change back tomorrow.

Sent something your way, danke nochmals aus der Steiermark :)

**Comment by MauriceNino** [comment]
Thank you, much appreciated :)

**Comment by MauriceNino** [comment]
:tada: This issue has been resolved in version 3.2.0

Please check the [changelog](https://github.com/MauriceNino/dashdot/blob/main/CHANGELOG.md) for more details.
